CONTRIBUTING.md

```diff
@@ -5,44 +5,61 @@ members — i.e., including you! It is our goal to build an inclusive
-
 ## Asking and answering questions about ASPECT
 
 For questions about ASPECT on all levels, please use the
 [ASPECT forum](https://community.geodynamics.org/c/aspect).
 Archived discussions from the inactive aspect-devel mailing list can be downloaded
 at [aspect-devel archives](https://geodynamics.org/pipermail/aspect-devel/).
 
-## Bug reports
+Please note that no one in the forum is paid or obliged to answer
+your questions. Everyone answering forum posts is volunteering
+their time to help make our community a friendly and helpful
+place to inhabit. Therefore, we ask that you keep your inquiries
+polite, take some effort to make them easily readable, and
+include all useful information as described below. If there is
+something you can test on your own, please test it first,
+before asking a question on the forum.
+
+There is no guarantee that we can or will help with your problem, but
+you are welcome to contact us and we will try to jointly improve
+ASPECT for your application case. Depending on the level of
+effort provided a thank you note or an acknowledgment in
+a paper is always appreciated. Keep in mind that for significant
+and scientifically creative contributions by a community member
+(but usually only then) a co-authorship on a publication
+is appropriate.
+
+## Reporting bugs and asking questions
 
 It is a great help to the community if you report any bugs that you
 may find. We keep track of all open issues related to ASPECT
 [here](https://github.com/geodynamics/aspect/issues).
 
-Please follow these simple instructions before opening a new bug report:
-
-- Do a search in the [list of open and closed issues](https://github.com/geodynamics/aspect/issues?q=is%3Aissue)
-  for a duplicate of your problem.
-- Do a search in the [ASPECT forum](https://community.geodynamics.org/c/aspect) for a
-  duplicate of your problem.
-- If you did not find an answer, open a new
-  [issue](https://github.com/geodynamics/aspect/issues/new) and explain your
-  problem in as much detail as possible.
-- Attach as much as possible of the following information to your issue:
-  - a minimal parameter file that reproduces the issue,
+Please follow these instructions before opening a new bug report or ask
+a question:
+
+- Make sure you have run your model in [DEBUG mode](https://aspect-documentation.readthedocs.io/en/latest/user/run-aspect/debug-mode.html). DEBUG mode reports many errors in greater details and may already solve your question.
+- Search in the [list of open and closed issues](https://github.com/geodynamics/aspect/issues?q=is%3Aissue)
+  for a duplicate of your question.
+- Search in the [ASPECT forum](https://community.geodynamics.org/c/aspect) for a
+  duplicate of your question.
+- If you did not find an answer in the previous searches, open a new question:
+  - If you suspect you have found a bug, open a new [issue](https://github.com/geodynamics/aspect/issues/new) 
+  and explain your problem as described below.
```

Comment:
did you do this line break without indentation on purpose? It looks a little off.

Reply:
That was an accident. Fixed now. I also modified some line breaks to make sure long links are on one line.